docker-compose.yml

```diff
@@ -22,6 +22,7 @@ services:
        context: .
        dockerfile: Dockerfile
      command: /app/docker-utils/run.sh
+     container_name: myapp
```

Comment:
Just a note.  I used to explicitly name my containers, until I found it made it difficult to spin up two instances of the codebase on the same machine.  In practicality, not something that happens too often, but found that the default names that docker picks are decent.

Reply:
Good call, is there is a way we can find the running container in the current folder?

I can update the `Makefile`

Reply:
Ah, I see, you were using the name in the Makefile.  I think attach is the only one were we need the container name. (logs and exec have a docker-compose equivalent form). 

I know you can get the container name with: "docker-compose ps app", but that also gives a bunch of useless header info.  I wonder if the --format  command can be used here... I'm still looking.

Reply:
ohh, interesting... I just found this.  I looks like it gets the container hash, which might be useful for the attach command:

```
docker-compose ps -q app
```

Reply:
Nice! Will update PR in a bit.